### PR TITLE
Add missing environment variables to CLSI README.

### PR DESCRIPTION
**Pull request by mans0954**
Requested change to https://github.com/overleaf/clsi/pull/213

I couldn't find a link to authoritative documentation as to what the `openout_any` env variable actually does. CC @GrahamMDouglas

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,17 +15,29 @@ The provided `Dockerfile` builds a docker image which has the docker command lin
...
+* `FILESTORE_PARALLEL_FILE_DOWNLOADS` - Number of parallel file downloads
+* `FILESTORE_PARALLEL_SQL_QUERY_LIMIT` - Number of parallel SQL queries
+* `LISTEN_ADDRESS` - The address for the RESTful service to listen on. Set to `0.0.0.0` to listen on all network interfaces
+* `PROCESS_LIFE_SPAN_LIMIT_MS` - Process life span limit in milliseconds
+* `SENTRY_DSN` - Sentry [Data Source Name](https://docs.sentry.io/product/sentry-basics/dsn-explainer/)
+* `SMOKE_TEST` - Whether to run smoke tests
+* `SQLITE_PATH` - Path to SQLite database
+* `SYNCTEX_BIN_HOST_PATH` - Path to SyncTeX binary
+* `TEXLIVE_IMAGE` - The TEXLIVE docker image to use for sibling containers, e.g. `gcr.io/overleaf-ops/texlive-full:2017.1`
+* `TEX_LIVE_IMAGE_NAME_OVERRIDE` - The name of the registry for the docker image e.g. `gcr.io/overleaf-ops`
```
Comment (aeaton-overleaf) [inline README.md:36]: Nitpick: "docker" => "Docker"

File: README.md [hunk 2]
```diff
@@ -15,17 +15,29 @@ The provided `Dockerfile` builds a docker image which has the docker command lin
...
+* `TEXLIVE_IMAGE_USER` - When using sibling containers, the user to run as in the TEXLIVE image. Defaults to `tex`
+* `TEXLIVE_OPENOUT_ANY` - Sets the `openout_any` environment variable for Tex Live
```
Comment (aeaton-overleaf) [inline README.md:38]: Nitpick: "Tex Live" and "TEXLIVE" => "TeX Live"

**Review by aeaton-overleaf — CHANGES_REQUESTED** [review]
Thanks! Just a couple of proof-reading nitpicks.

**Review by aeaton-overleaf — APPROVED** [review]

**Comment by mans0954** [comment]
> 'openout_any' is discussed in the Web2C documentation, this section: http://tug.org/texinfohtml/web2c.html#tex-invocation. Look for the sentence starting "TeX can write output files, via the \openout primitive..."

Thanks! I've linked to that documentation in the README.

**Pull request merged**